fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckScaffold.kt

```diff
@@ -86,18 +89,23 @@ fun ReviewQualityCheckScaffold(
 }
 
 @Composable
-private fun Header() {
+private fun Header(modifier: Modifier) {
```

Comment:
We don't need to expose a modifier here since this is a private composable. We can directly set the Modfier on Row.